### PR TITLE
Moved tools section to left on large devices (Bootstrap xl)

### LINKED ISSUE #13077: support for large monitors
### Steps to reproduce
1. try clicking button far on the rught
2. see the image
screenshot 9

3.

### Expected behaviour
move buttons to the right

### Actual behaviour
buttons are on the far left

### Server configuration
**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**phpMyAdmin version:**

### Client configuration
**Browser:**

**Operating system:**

### PR DESCRIPTION
**Pull request by rajat315315**
Signed-off-by: Rajat Jain <rajatjain.ix@gmail.com>

### Description

Moved the tools section to the left side for large monitors.

Fixes #13077

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by ibennetch** [comment]
Sorry I'm having trouble seeing it, which tools area does this affect?

**Comment by rajat315315** [comment]
The tools section containing Profile, Edit Inline, Edit, Refresh, just below SQL query

**Comment by rajat315315** [comment]
Any review or merge?

**Comment by williamdes** [comment]
Hello @rajat315315  
I think you can take some inspiration from the discussions in https://github.com/phpmyadmin/phpmyadmin/pull/15715#issuecomment-619351079

**Comment by rajat315315** [comment]
Ok, Understood!
I will look for media queries now..

**Comment by rajat315315** [comment]
Done..

File: themes/bootstrap/scss/_common.scss [hunk 1]
```diff
@@ -3801,3 +3801,10 @@ body .ui-dialog {
 .ui-timepicker-div dl .ui_tpicker_timezone select {
   margin-left: 50px;
 }
+
+// Extra large devices (large desktops, 1200px and up)
+@media (min-width: 1200px) {
```
Comment (williamdes) [inline themes/bootstrap/scss/_common.scss:3806]: I think you can use bootstrap variables from https://getbootstrap.com/docs/4.0/layout/grid/#variables

**Review by williamdes — APPROVED** [review]
Near perfect, good work
One last change required :)

File: themes/bootstrap/scss/_common.scss (hunk 1 above)
Comment (rajat315315) [reply themes/bootstrap/scss/_common.scss]: Done!

File: themes/bootstrap/scss/_common.scss [hunk 2]
```diff
@@ -3801,3 +3801,10 @@ body .ui-dialog {
 .ui-timepicker-div dl .ui_tpicker_timezone select {
   margin-left: 50px;
 }
+
+// Extra large devices (large desktops, 1200px and up)
+@include media-breakpoint-up(xl){
+  div.tools {
+    text-align: left;
```
Comment (williamdes) [inline themes/bootstrap/scss/_common.scss:3808]:
```suggestion
    text-align: #{$left};
```
Maybe you need to use this instead to support RTL screens ?
Please try a RTL language and let me know :)

Comment (rajat315315) [reply themes/bootstrap/scss/_common.scss:3808]: Ohh.. Never thought of RTL..
Will look into it.

**Review by williamdes — APPROVED** [review]

**Pull request merged**